scripts/lis_hyperv_platform/main.ps1

```diff
@@ -21,11 +21,13 @@ param(
     [String] $SharedStoragePath,
     [String] $ShareUser,
     [String] $SharePassword,
-    [String] $IdRSAPub
+    [String] $IdRSAPub,
+    [String] $LisaTestDependencies
 )
 
 $ErrorActionPreference = "Stop"
 
+$global:LISA_REL_PATH = ".\lis-test\WS2012R2\lisa"
```

Comment:
just leave $LISA_REL_PATH, we do not need an outside global variable, only a local global

Reply:
LISA_FOLDER = ".\lis-test"
LISA_REL_PATH = "${LISA_FOLDER}\WS2012R2\lisa"